docs/changelog.rst

```diff
@@ -6,6 +6,8 @@ Changelog
 `Unreleased`_
 -------------
 
+- Add ``port`` parameter to ``from_uri()`` method. `#706`_
```

Comment:
Could you please add `.. _#706: https://github.com/schemathesis/schemathesis/issues/706` string to the end of this file where other links are placed? It will make the rendered changelog point to the proper issue location.